src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -1319,7 +1319,7 @@ public async Task PositionalPatternFirstPosition()
             await VerifyItemExistsAsync(AddUsingDirectives("using System;", AddInsideMethod(content)), @"System");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/40015"), Trait(Traits.Feature, Traits.Features.Completion)]
```

Comment:
Should this also have a prototype comment?

Reply:
Only needed when there is no tracking issue.

---
In reply to: [352880444](https://github.com/dotnet/roslyn/pull/39913#discussion_r352880444) [](ancestors = 352880444)